### ISSUE TITLE
Feature request: percentage expressed

### ISSUE THREAD

**Issue by GBeattie** [neurorestore/Libra#28]
Hey, just started using this package today, very good so far!

I've noticed in the issues a few feature requests (mostly the ability to regress covariates, but also a one vs all approach analogous to FindAllMarkers, both of which I would be very interested in). I also noticed you are talking about releasing another version which may incorporate some of these suggestions. 

One thing I think would be very useful, again analogous to Seurat, is to include % of cells expressing each gene by group in the output. Usefully there's a hard filter in run_de (min_features), but with the higher sensitivity of pseudobulk many of the bigger logFCs are in lowly expressed genes, so a readout such as pct.1/pct.2 would be of great utility.

Thanks for considering any of these additions!

All the best,
Gordon

**Comment by jordansquair** [comment]
Yes definitely will be a good addition to have this. Thanks Gordon!
